src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/alignment/AssemblyContigWithFineTunedAlignments.java

```diff
@@ -29,20 +30,29 @@
      */
     private final String saTAGForGoodMappingToNonCanonicalChromosome;
 
-    public AssemblyContigWithFineTunedAlignments(final AlignedContig contig) {
-        this(contig, emptyInsertionMappings, null);
+    public AssemblyContigWithFineTunedAlignments(@Nonnull final AlignedContig contig) {
+        this(Utils.nonNull(contig), emptyInsertionMappings, NO_GOOD_MAPPING_TO_NON_CANONICAL_CHROMOSOME);
     }
 
-    public AssemblyContigWithFineTunedAlignments(final AlignedContig contig,
-                                                 final List<String> insertionMappings,
-                                                 final String saTAGForGoodMappingToNonCanonicalChromosome) {
-        this.contig = contig;
-        this.insertionMappings = Utils.nonNull(insertionMappings);
+    // {@code goodMappingToNonCanonicalChromosome} could be null
+    public AssemblyContigWithFineTunedAlignments(@Nonnull final AlignedContig contig,
+                                                 @Nonnull final List<String> insertionMappings,
+                                                 final AlignmentInterval goodMappingToNonCanonicalChromosome) {
+        this(Utils.nonNull(contig), Utils.nonNull(insertionMappings),
```

Comment:
Do you need both calls to `Utils.nonNull` and the annotations in the method signature?

Reply:
maybe I'm just paranoid.... removed the annotation